src/routes/safe/components/Apps/AddAppForm/SubmitButtonStatus.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useFormState } from 'react-final-form'
+
+import { SafeApp } from 'src/routes/safe/components/Apps/types'
+
+interface SubmitButtonStatusProps {
```

Comment:
is there a reason to use an interface instead of a type? 
As far I understand we should use type unless it could be extended in somewhere else.

Reply:
not sure, I've been using interface lately.

Do we have a decision made around that or a standard?

https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#function-components